build.sbt

```diff
@@ -671,6 +671,9 @@ lazy val http4s = projectMatrix
         Dependencies.Alloy.`protocol-tests` % Test
       )
     },
+    Compile / allowedNamespaces := Seq(
+      "smithy4s.example.guides.auth"
+    ),
```

Comment:
**nitpick:** I thought `bootstrapped` was the only module that generates code, but apparently not.

however, this should probably be scoped to `Test` instead of `Compile`, just like the other codegen-specific settings here.